README.md

```diff
@@ -222,5 +222,6 @@ curl -H "x-retry-test-id: 1d05c20627844214a9ff7cbcf696317d" "http://localhost:91
 | ----------------------- | ---
 | return-X                | Testbench will fail with HTTP code provided for `X`, e.g. return-503 returns a 503
 | return-X-after-YK       | Testbench will return X after YKiB of uploaded data
+| return-200-with-YB      | Testbench will return 200 with a response of Y bytes from object metadata JSON on resumable upload completion.
```

Comment:
As discussed, I feel like the name is slightly unclear. IIUC this is causing a broken stream after Y bytes-- so let's call it that instead of return-200

Reply:
Because of the risk of confusing this "broken stream of json metadata" with a "broken stream of actual data download", may I suggest something like `return-200-with-truncated-json`?

Reply:
I decided to go with `return-broken-stream-final-chunk-after-YB`